src/common/components/feedback-form/FeedbackForm.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import ChipSelection from '../chip-selection/ChipSelection';
+import ContactInputField from '../contact-input-field/ContactInputField';
+import Textarea from '../textarea/Textarea';
+import { FeedbackFormConfig } from './configs/FeedbackForm.config';
+import confirmLogo from '../../../assets/icons/confirm-icon.svg';
+
+const FeedbackForm = () => {
+  const [showForm, setShowForm] = useState<boolean>(true);
+  const { t } = useTranslation(['feedback_form', 'common']);
+
+  const {
+    handleSubmit,
+    control,
+    reset,
+    formState: { errors },
+  } = useForm({
+    mode: 'onChange',
+    reValidateMode: 'onChange',
+  });
+
+  const onSaveFeedback = () => {
+    console.log('Not yet implemented');
+    reset();
+    setShowForm(false);
```

Comment:
Ar trebui si erorile afisate undeva cand se va realiza Mutation-ul.
Poate ar mai trebui o sectiune, la fel ca cea de succes, in care sa afisezi eroarea de la BE / Network

Reply:
Nu ar fi mai ok un toast decat o sectiune? Si cred ca ar fi mai usor sa se adauge erorile cand se va face mutatia.